srcpkgs/Dwarf-Therapist/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'Dwarf-Therapist'
+pkgname=Dwarf-Therapist
+version=41.0.3
+revision=1
+build_style=cmake
+makedepends="qt5-declarative-devel libcap-devel hicolor-icon-theme"
+depends="dwarffortress-0.44.12_2"
```

Comment:
does it really have to depend on a specific version of the package? This will break when dwarffortress gets updated.

Reply:
A dwarffortress version update will almost certainly require a Dwarf Therapist update, but its not guaranteed, and it's unfortunate the revision can't be elided. Should the version dependency just be deleted and left to the sensibilities of the user?